data/markdown/partials/solution/content-management/developer-collection.md

```diff
@@ -0,0 +1,53 @@
+---
+solution: ['content-management']
+product: ['experience-manager']
+title: 'Developer Collection'
+description: 'Created by developers, for developers, this suite of tools allows your team to automate builds and deployments, easily compare and merge databases, and collaborate more efficiently.'
+---
+
+## Introduction
+The Sitecore Developer Collection was created by developers, for developers, to eliminate down time and headaches when working with Sitecore. The suite of tools allows your team to automate builds and deployments, easily compare and merge databases, and collaborate more efficiently. This incredibly feature-rich tool set is highly adaptable and easily customized to fit your project.
+
+## Sitecore CLI
+The new content serialization tooling combines the best of both TDS & Unicorn, making it much easier for you to script your content changes and move them between your different environments as part of your deployment process. Sitecore CLI provides a command line interface to interact with your Sitecore instance. This command line provides technical team members with automation capabilities and the ability to work at the command line directly with their Sitecore instance.
+
+* [Download Sitecore CLI plugin](https://dev.sitecore.net/Downloads/Sitecore_CLI.aspx)
+* [Documentation](https://doc.sitecore.com/en/developers/101/developer-tools/sitecore-command-line-interface.html)
+
```

Comment:
I think we need to separate the CLI from Sitecore Content Serialisation here as they're separate products. 

The CLI does work with SCS, but also Indexing, Publishing etc and I believe there are plans for further plugins, so it should have a section in it's own right.

Reply:
Hey @robearlam , I think I'm not clear on the request here. Are you saying you want to add a new SCS section on this page? Or that there is a reference in this page that you want removed? Or both?

(If there are no serious issues, we need to get this page published and then iterate on it... right now it is missing entirely)

Reply:
@markvanaalst explained this to me, I'm going to do a change now and make this more generic about the overall CLI and then merge up to main.